flake.nix

```diff
@@ -224,6 +224,8 @@
                 };
                 "infra/makemake" = toplevel self.nixosConfigurations.makemake;
                 "infra/overview" = self.packages.${system}.overview;
+                # fake derivation for flake check
+                "infra/templates" = pkgs.writeText "dummy" (lib.strings.toJSON classic'.templates.project);
```

Comment:
Flake checks require a derviation. This is the best way I found to do this so far, so I'm happy for any suggestions ...